tests/test_tezos_interop.re

```diff
@@ -24,7 +24,7 @@ describe("key", ({test, _}) => {
     ).
       toBe(
       // TODO: proper equals
-      ~equals=(==),
```

Comment:
Perhaps after the review is complete and the PR is getting merged, the TODO comment can be removed

Reply:
removed